src/Core/BufferUpdate.re

```diff
@@ -7,7 +7,7 @@ type t = {
   id: int,
   startLine: Index.t,
   endLine: Index.t,
-  lines: array(string),
+  lines: [@opaque] array(string),
```

Comment:
Thanks for adding this! The buffer updates could be noisy in the logs, especially for large files